src/api/app/views/webui2/webui/users/subscriptions/index.html.haml

```diff
@@ -1,32 +1,32 @@
 - @pagetitle = 'Notifications'
 
-.card
-  .card-header
+.card.mb-2
+  .card-header.d-flex.justify-content-between
     %h5 Notifications
   .card-body
     = form_tag(user_notifications_path, method: :put, id: 'notification_form') do
       - unless @groups_users.empty?
-        .card-block
-          %h3 Groups
-          %p You will receive emails from the checked groups
-          - @groups_users.each do |group_user|
-            .custom-control.custom-checkbox.custom-control-inline
-              = check_box_tag group_user.group, '1', group_user.email, class: 'custom-control-input', id: "checkbox-#{group_user.group}"
-              = label_tag group_user.group.title, nil, class: 'custom-control-label', for: "checkbox-#{group_user.group}"
-        %br
-      .card-block
-        %h3 Events
-        %p Choose from which events you want to get an email
-        %ul.list-group.list-group-flush
-        #subscriptions-form
-          = render partial: 'webui/subscriptions/subscriptions_form'
+        .card.mb-3
+          .card-body
+            %h4.card-title Groups
+            %p You will receive emails from the checked groups
+            - @groups_users.each do |group_user|
+              .custom-control.custom-checkbox.custom-control-inline
+                = check_box_tag group_user.group, '1', group_user.email, class: 'custom-control-input', id: "checkbox-#{group_user.group}"
+                = label_tag group_user.group.title, nil, class: 'custom-control-label', for: "checkbox-#{group_user.group}"
+      .card.mb-3
         .card-body
-          = submit_tag 'Update', class: 'btn btn-primary'
-          = link_to('Reset to default', user_notifications_path(default_form: true), remote: true)
+          %h4.card-title Events
+          %p Choose from which events you want to get an email
+          %ul.list-group.list-group-flush
+          #subscriptions-form
```

Comment:
The indentation is wrong here. Should the `div` be outside of the `ul`?

![wrong-indentation](https://user-images.githubusercontent.com/1102934/53483604-1bea2500-3a82-11e9-851f-5eaae48e0dd4.png)

Reply:
Something not related to the changes of this pull request, but I will have a look.

Reply:
Almost every line of the view is changed somehow (indentation, new CSS classes, etc...). What would we need to do to have this related to this PR?

Reply:
I preserved the relative indentation of the ` %ul.list-group.list-group-flush` and `#subscriptions-form` lines as it was before, because this PR was not intended to fix it.

Reply:
Done. I've copied the work in progress from #7036, so now that Boostrap card css clases are used, the `ul` is no longer needed.